packages/@aws-cdk/aws-codedeploy/lib/deployment-group.ts

```diff
@@ -143,6 +174,20 @@ export interface ServerDeploymentGroupProps {
    * @see https://docs.aws.amazon.com/codedeploy/latest/userguide/codedeploy-agent-operations-install.html
    */
   installAgent?: boolean;
+
+  /**
+   * All EC2 instances matching the given set of tags will be added to this Deployment Group.
```

Comment:
Can we make the comments more clear - instances are expanded by tag during deployment only.

Reply:
Good call. Will change.